### DISCUSSION TITLE
How to change the projection plane of pole figures?

### DISCUSSION THREAD

**Discussion by zstar-kkk** [mtex-toolbox/mtex#2184] [Ask Anything] [answered]
Hi MTEX community,
  I need help plotting the pole figures with the projection plane xz . Can someone assist me with the code for this?
  Thanks in advance for your help.

**Comment by ralfHielscher** [comment, marked as answer]
This is only possible with MTEX 6.0. There you can do
```matlab
ori = orientation.rand(cs);
h = Miller(1,0,0,cs);

% define the plotting convention
pC = plottingConvention(xvector, yvector)

% do the plotting
plotPDF(ori,h,pC)
```

Ralf.

**Reply by zstar-kkk** [reply]
Thank you!
